gen/gen.go

```diff
@@ -386,23 +325,35 @@ func (g *Generator) addThirdPartyImport(pkg string) {
 
 // Write writes Go code from the parsed schemas.
 func (g *Generator) Write(w io.Writer) error {
-	parsed, err := template.New("out").Parse(outputTemplate)
+	parsed, err := template.New("out").
+		Funcs(template.FuncMap{
+			"kebab":      strcase.ToKebab,
+			"upperCamel": strcase.ToPascal,
+			"camel":      strcase.ToCamel,
+			"snake":      strcase.ToSnake,
+		}).
+		Parse(g.template)
 	if err != nil {
 		return err
 	}
 
+	imports := g.imports
+	if len(g.thirdPartyImports) > 0 {
+		imports = append(imports, "")
```

Comment:
This seems a little odd to me, having an empty import.

Reply:
fixed by using https://pkg.go.dev/golang.org/x/tools/imports#Process